try {
    alerrt('helltoyou')
} catch (error) {
    console.log('error');
}